spec/asyncapi.md

```diff
@@ -1,10 +1,10 @@
 # AsyncAPI Specification
 
-#### Attribution
+## Attribution
 
 Part of this content has been taken from the great work done by the folks at the [OpenAPI Initiative](https://openapis.org).
 
-#### Version 3.0.0
+### Version 3.0.0
```

Comment:
I remember having some issues with the right panel navigation in the rendered website whenever we changed that. Not saying this will break anything but not sure either if this will happen 🤔 Just double checking with @derberg just in case you know if this will be rendered properly. 

For the reference, I mean this panel 
![Google Chrome_EVccZsxA](https://github.com/asyncapi/spec/assets/1083296/9fd70c26-a233-4493-a905-47d34d4d7fdb)

Reply:
I don't recall any issues really, as h3 all should be good I think